### PR TITLE
Ahinojo open positions

### PR DESCRIPTION
**Pull request by abrahamhinojo**
open positions

### CODE REVIEW

**Comment by echavezNS** [comment]
Can you pull master to this branch? In order to see if there is any conflicts

**Comment by echavezNS** [comment]
Is this how the page looks in your local? 

image

File: views/pages/position.jade [hunk 1]
```diff
@@ -1,4 +1,47 @@
...
+
+      div(class="our-hiring-process-icons" style="background-image: url('images/icons.png')")
+        div(class="our-hiring-process-icons-item our-hiring-process-icons-item-1" style="background-image: url('images/icons.png')")
+        div(class="our-hiring-process-icons-item our-hiring-process-icons-item-2" style="background-image: url('images/icons.png')")
+        div(class="our-hiring-process-icons-item our-hiring-process-icons-item-3" style="background-image: url('images/icons.png')")
+        div(class="our-hiring-process-icons-item our-hiring-process-icons-item-4" style="background-image: url('images/icons.png')")
+        div(class="our-hiring-process-icons-item our-hiring-process-icons-item-5" style="background-image: url('images/icons.png')")
+        div(class="our-hiring-process-icons-item our-hiring-process-icons-item-6" style="background-image: url('images/icons.png')")
+        div(class="our-hiring-process-icons-item our-hiring-process-icons-item-7" style="background-image: url('images/icons.png')")
+        div(class="our-hiring-process-icons-item our-hiring-process-icons-item-8" style="background-image: url('images/icons.png')")
```
Comment (abrahamhinojo) [inline views/pages/position.jade]: check

File: views/pages/position.jade [hunk 2]
```diff
@@ -1,4 +1,47 @@
...
+      p El proceso no es sencillo, debemos confesar, pero una vez que formes parte del team, te darás cuenta que habrá valido la pena.
+
+  section.requirements
+    div.wrapper
+      div.requirements-section.profiles
+        div.profiles-container
+          span OP Profiles &
+          span.bold Requirements
+      div.requirements-section.github
+        a(class="github-link" style="background-image: url('images/github.png')")
```
Comment (abrahamhinojo) [inline views/pages/position.jade]: check

**Pull request merged**